### PR TITLE
Trunc CSV

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: printer/dataset_printer.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func (p *DataSetPrinter) SetOutCsv(filename string) {
 	if p.fd != nil {
 		p.UnsetOutCsv()
 	}
-	fd, err := os.OpenFile(filename, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0600)
+	fd, err := os.OpenFile(filename, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0600)
```
Comment (darionyaphet) [inline printer/dataset_printer.go:38]: truncate regular writable file when opened. So make sure the file won't be too large?

Comment (CPWstatic) [inline printer/dataset_printer.go:38]: What happens to `Append`?

Comment (yixinglu) [reply printer/dataset_printer.go:38]: Avoid to save different query results to same csv file. Use TRUNC mode to override the old result.

Comment (yixinglu) [reply printer/dataset_printer.go:38]: See above comment.

**Review by darionyaphet — APPROVED** [review]

**Pull request closed without merging**